lib/phobos_db_checkpoint/version.rb

```diff
@@ -1,3 +1,3 @@
 module PhobosDBCheckpoint
-  VERSION = '3.1.0'
+  VERSION = '3.2.0'
```

Comment:
In terms of semantic versioning, I guess removing `pool_size` option is breaking backward compatibility, right? On the other hand, the lib will still work (not crash), but will ignore the option. 🤔

Reply:
We will print deprecation warnings, I think it's not a breaking change.